src/node_errors.h

```diff
@@ -163,6 +166,7 @@ ERRORS_WITH_CODE(V)
   V(ERR_INVALID_STATE, "Invalid state")                                        \
   V(ERR_INVALID_THIS, "Value of \"this\" is the wrong type")                   \
   V(ERR_INVALID_TRANSFER_OBJECT, "Found invalid object in transferList")       \
+  V(ERR_INVALID_URL_SCHEME, "The URL must be of scheme file:")                 \
```

Comment:
the name and the error message creates confusion.

Reply:
I got from: 
https://github.com/nodejs/node/blob/cd6bd3f0fb135d3ef7cc847d62205c6f0d7a47c8/lib/internal/errors.js#L1381-L1390

And:

https://github.com/nodejs/node/blob/cd6bd3f0fb135d3ef7cc847d62205c6f0d7a47c8/lib/internal/url.js#L1359-L1360C29

What I can do is remove this default message and pass the message when I create the error.